webpack.config.js

```diff
@@ -177,10 +178,11 @@ var config = {
         : path.join(__dirname, 'src/sentry/integration-docs/_platforms.json')
     },
     modules: [path.join(__dirname, staticPrefix), 'node_modules'],
-    extensions: ['*', '.jsx', '.js', '.json']
+    extensions: ['.less', '.jsx', '.js', '.json']
   },
   output: {
     path: distPath,
+    publicPath: '/_static/sentry/dist/',
```

Comment:
Nope, I think this was added in the old styleguide when we wanted to codesplit, so seems good to remove.